src/info.json

```diff
@@ -0,0 +1,137 @@
+{
+   "wisconsin": {
+      "name": "Wisconsin Non-Metallic Mining",
+      "url": "https://services.arcgis.com",
+      "api_url": "https://services.arcgis.com/jDGuO8tYggdCCnUJ/arcgis/rest/services/Nonmetallic_and_Potential_frac_sand_mine_proposals_in_West_Central_Wisconsin/FeatureServer/0/",
+      "extents": {
+        "NE_lat":  44.08758502824518,
```

Comment:
This is looking awesome. Great work. Are there maybe standard parameter formats from the Leaflet API for bounds? 

https://leafletjs.com/reference-1.4.0.html#bounds

This seems to show:

```js
[
    [40.712, -74.227],
    [40.774, -74.125]
]
```

...noting that each pair can also be a `LatLng` object.

Reply:
Okay 👍